readme.md

```diff
@@ -273,3 +273,29 @@ final Response response = serviceFactory.get(request);
 ```
 
 
+## How do I use Address Submission?
+
+Add the Address Submission dependency:
+```xml
+<dependency>
+	<groupId>com.afrigis.services</groupId>
+	<artifactId>addresssubmission</artifactId>
+	<version>3.0.1</version>
+</dependency>
+```
+
+
+```java
+import com.afrigis.services.AfriGISServices;
+import com.afrigis.services.ServiceCallFactory;
+import com.afrigis.services.geocode.AddressRequest;
+import com.afrigis.services.geocode.AddressResponse;
+
+ServiceCallFactory serviceFactory = AfriGISServices.instance("Key","Secret");
+AddressSubmissionRequest addressSubParams = new AddressSubmissionRequest("KEY", "Address", "AdditionalInfomation");
```

Comment:
What is the `KEY` in this context?

Maybe just add a comment to make it clear it is your SaaS key?